--- vbcc/datatypes/dtgen.c.orig	2013-04-24 00:45:50.000000000 +0200
+++ vbcc/datatypes/dtgen.c	2015-04-28 10:54:11.828847716 +0200
@@ -130,23 +130,14 @@
 int askyn(char *def)
 {
   char in[8];
-  do{
-    printf("Type y or n [%s]: ",def);
-    fflush(stdout);
-    fgets(in,sizeof(in),stdin);
-    if(*in=='\n') strcpy(in,def);
-  }while(*in!='y'&&*in!='n');
+  strcpy(in,def);
   return *in=='y';
 }
 
 char *asktype(char *def)
 {
   char *in=mymalloc(128);
-  printf("Enter that type[%s]: ",def);
-  fflush(stdout);
-  fgets(in,127,stdin);
-  if(in[strlen(in)-1]=='\n') in[strlen(in)-1]=0;
-  if(!*in) strcpy(in,def);
+  strcpy(in,def);
   return in;
 }
 
